README.md

```diff
@@ -33,19 +33,14 @@ This plugin is compatible with CSI versions [v1.2.0](https://github.com/containe
 The following table captures the compatibility matrix of the core persistent disk driver binary
 `gke.gcr.io/gcp-compute-persistent-disk-csi-driver`
 
-| GCE PD CSI Driver\Kubernetes Version | 1.15 | 1.16  | 1.17+ |
-|--------------------------------------|------|-------|-------|
-| v0.2.x (alpha)                       | no   | no    | no    |
-| v0.3.x (beta)                        | yes  | yes   | yes   |
-| v0.4.x (beta)                        | yes  | yes   | yes   |
-| v0.5.x (beta)                        | yes  | yes   | yes   |
-| v0.6.x (beta)                        | yes  | yes   | yes   |
-| v0.7.x (beta)                        | yes  | yes   | yes   |
-| v1.0.x (ga)                          | yes  | yes   | yes   |
-| dev                                  | yes  | yes   | yes   |
+| GCE PD CSI Driver\Kubernetes Version | 1.17+ |
+|--------------------------------------|-------|
+| v0.7.x (beta)                        | yes   |
```

Comment:
so we want to remove the older version information?

Reply:
yes, it's just noise I think, especially since I also removed the older overlays as well.